tests/testsuite/build_script_extra_link_arg.rs

```diff
@@ -403,3 +403,39 @@ fn build_script_extra_link_arg_examples() {
         )
         .run();
 }
+
+#[cargo_test]
+fn issue_12663() {
```

Comment:
Generally it is best to give the test a descriptive name instead of issue numbers, and a comment explaining what the test does (the comment can include a full URL to the issue).